File: Basic Coding Problems/01 Fundamentals/005 Write a program to enter the length and width of a rectangle and find its area/C/rectangle_area.c
#include <stdio.h>

int main() {
    int length, width, area;

    printf("Enter the length of the rectangle: ");
    scanf("%d", &length);

    printf("Enter the width of the rectangle: ");
    scanf("%d", &width);

    area = length * width;

    printf("The area of the rectangle is: %d\n", area);

    return 0;
}

File: Basic Coding Problems/01 Fundamentals/005 Write a program to enter the length and width of a rectangle and find its area/Java/rectangle_area.java
import java.util.Scanner;

public class rectangle_area {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Get the length and width of the rectangle from the user.
        System.out.print("Enter the length of the rectangle: ");
        int length = scanner.nextInt();
        System.out.print("Enter the width of the rectangle: ");
        int width = scanner.nextInt();

        // Calculate the area of the rectangle.
        int area = length * width;

        // Print the area of the rectangle.
        System.out.println("The area of the rectangle is: " + area);
    }
}